### PR TITLE
test(cli): snapshot new da dx changes

### PR DESCRIPTION
**Pull request by millsp**
closes https://github.com/prisma/team-orm/issues/867

### CODE REVIEW

File: packages/cli/src/__tests__/fixtures/custom-output-npm/package-lock.json
Comment (Jolg42) [inline packages/cli/src/__tests__/fixtures/custom-output-npm/package-lock.json:1]: It's weird to me that this file is empty 🤔

Comment (Jolg42) [reply packages/cli/src/__tests__/fixtures/custom-output-npm/package-lock.json:1]: So it's just for the "detection" logic to find it?

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 1]
```diff
@@ -82,6 +82,189 @@ describe('using cli', () => {
...
+
+        Prisma Client has been generated to a custom path:
+
+        1. Make it a dependency of your project
+        \`\`\`
+        # adapt this relative path if needed
+        pnpm add db@link:./generated/client
+        \`\`\`
+
+        More information: https://pris.ly/d/custom-output
```
Comment (Jolg42) [inline packages/cli/src/__tests__/commands/Generate.test.ts:191]: `https://pris.ly/d/custom-output` does not exist? I get a redirect to the landing page (`https://www.prisma.io/`)

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 2]
```diff
@@ -82,6 +82,189 @@ describe('using cli', () => {
...
+        Prisma schema loaded from prisma/schema.prisma
+
+        ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
+
+        Prisma Client has been generated to a custom path:
+
+        1. Make it a dependency of your project
+        \`\`\`
+        # adapt this relative path if needed
+        yarn add db@link:./generated/client
```
Comment (Jolg42) [inline packages/cli/src/__tests__/commands/Generate.test.ts:104]: @millsp I guess you tried this command already?

File: packages/cli/src/__tests__/commands/Generate.test.ts [hunk 3]
```diff
@@ -82,6 +82,189 @@ describe('using cli', () => {
...
+        Prisma schema loaded from prisma/schema.prisma
+
+        ✔ Generated Prisma Client (v0.0.0) to ./generated/client in XXXms
+
+        Prisma Client has been generated to a custom path:
+
+        1. Make it a dependency of your project
+        \`\`\`
+        # adapt this relative path if needed
+        npm i db@./generated/client
```
Comment (Jolg42) [inline packages/cli/src/__tests__/commands/Generate.test.ts:146]: @millsp I guess you tried this command already?

**Review by Jolg42 — APPROVED** [review]

**Review by Jolg42 — APPROVED** [review]
See comments

File: packages/cli/src/__tests__/commands/Generate.test.ts (hunk 3 above)
Comment (millsp) [reply packages/cli/src/__tests__/commands/Generate.test.ts:146]: Yes

File: packages/cli/src/__tests__/commands/Generate.test.ts (hunk 2 above)
Comment (millsp) [reply packages/cli/src/__tests__/commands/Generate.test.ts:104]: Yes

File: packages/cli/src/__tests__/fixtures/custom-output-npm/package-lock.json
Comment (millsp) [reply packages/cli/src/__tests__/fixtures/custom-output-npm/package-lock.json:1]: Yes

Comment (millsp) [reply packages/cli/src/__tests__/fixtures/custom-output-npm/package-lock.json:1]: It is then detected by `@antfu/ni`

File: packages/cli/src/__tests__/commands/Generate.test.ts (hunk 1 above)
Comment (millsp) [reply packages/cli/src/__tests__/commands/Generate.test.ts:191]: Yes, this is work that needs to be done by our DAs

**Pull request merged**